Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -1058,10 +1058,9 @@ inline void AXObjectCache::setForceDeferredSpellChecking(bool shouldForce)
     gForceDeferredSpellChecking = shouldForce;
 }
 
-class AXAttributeCacheEnabler final
-{
+class AXAttributeCacheEnabler final {
 public:
-    explicit AXAttributeCacheEnabler(AXObjectCache *cache);
+    explicit AXAttributeCacheEnabler(AXObjectCache *);
```

Comment:
To avoid conflict with https://github.com/WebKit/WebKit/pull/49218/files, I'd recommend removing this part of the change. I think @minorninth fixed this when he moved `AXAttributeCacheEnabler` to its own header 👍